### PR TITLE
Move test files

### PR DESCRIPTION
**Pull request by jdebacker**
This PR moves the test files out of the `ogcore` directory.  The hope is that doing this will allow for a smaller `ogcore` package to be created and distributed via PyPi.

### CODE REVIEW

**Comment by jdebacker** [comment]
@rickecon This PR is ready for review.

File: .github/workflows/build_and_test.yml [hunk 1]
```diff
@@ -14,7 +14,7 @@ jobs:
     runs-on: ubuntu-latest
     strategy:
       matrix:
-        python-version: [3.8, 3.9]
+        python-version: [3.8, 3.9, '3.10']
```
[the remark this answers is not in the document]
Comment (rickecon) [reply .github/workflows/build_and_test.yml:17]: @jdebacker. This is a great PR. I have reviewed all of it, and it looks ready to go. Thanks for doing this nice cleanup that will allow for a streamlined and efficient package. I am ready to merge this as soon as you respond to my very small comment on the single quotes around 3.10 in line 17 of `build_and_test.yml`.

**Comment by rickecon** [comment]
@jdebacker. This is a great PR. I have reviewed all of it, and it looks ready to go. Thanks for doing this nice cleanup that will allow for a streamlined and efficient package. I am ready to merge this as soon as you respond to my very small comment on the single quotes around 3.10 in line 17 of `build_and_test.yml`.

File: .github/workflows/build_and_test.yml (hunk 1 above)
[the remark this answers is not in the document]
Comment (jdebacker) [reply .github/workflows/build_and_test.yml:17]: @rickecon If 3.10 is not in quotes it is interpreted as 3.1, which is not what we want.

**Pull request merged**